packages/nimble-components/src/utilities/tests/storybook.ts

```diff
@@ -161,3 +161,12 @@ export const disableStorybookZoomTransform = `
     }
 </style>
 `;
+
+export const argsTableCategory = {
+    attributes: 'Attributes',
+    events: 'Events',
+    methods: 'Methods',
+    slots: 'Slots'
+} as const;
+export type SpinnerAppearance =
+    (typeof argsTableCategory)[keyof typeof argsTableCategory];
```

Comment:
What's going on here? Should either remove this bit (notice the `SpinnerAppearance`) definition and just export an object or if we want to represent as an enum we need to move to a types.ts file and fix the naming. I'd probably just go with `ApiCategory` since it's not really about the args table but about the component apis.

Reply:
Oops, mindless copy-paste bug, I simplified as suggested.